docs/install/ubuntu_setup.md

```diff
@@ -137,12 +137,33 @@ It is recommended that you review the general [build from source](build_from_sou
 
 On Ubuntu versions 16.04 or later, you need the following dependencies:
 
-**Step 1:** Install build tools and git.
+**Step 1:** Install prerequisite packages.
 ```bash
     sudo apt-get update
-    sudo apt-get install -y build-essential git
+    sudo apt-get install -y build-essential git ninja-build ccache
```

Comment:
Again, even though ccache is good, I am not in favor of adding something optional as a prerequisite for installation. But I am open to other opinions.

Reply:
It really speeds up compilation and CI uses it. I think is strongly recommeded, I don't see the problem with this.

Reply:
Fine, if it's been widely adopted across projects.